### ISSUE TITLE
Can year increments be based on years in the excel and not just increase by 1

### ISSUE THREAD

**Issue by gabbybamps** [SjoerdTilmans/sjvisualizer#23]
Can the increments in the years in the animation be based on the next year in line in the excel and not just increase by 1 always. 

For example if the years in my excel are 1990, 1995, 1998, 2004, 2007.... , is it possible to introduce a parameter to set to determine which is the next year (in this case move from 1990 to 1995 to 1998 to 2004 to 2007 etc.) for the animation instead of moving from (1990 to 1991 to 1992 to 1993 etc) which may have no data and will just increase the time for the animation.

In other words, can the year increment factor be made dynamic and not set to 1.
